frameworks/Java/servicetalk/pom.xml

```diff
@@ -0,0 +1,89 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.techempower</groupId>
+    <artifactId>servicetalk</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-serialization-api</artifactId>
+            <version>0.20.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-http-api</artifactId>
+            <version>0.20.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-http-netty</artifactId>
+            <version>0.20.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-concurrent-api</artifactId>
+            <version>0.20.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-data-jackson</artifactId>
+            <version>0.20.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-transport-netty-internal</artifactId>
+            <version>0.20.0</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <finalName>servicetalk</finalName>
+        <plugins>
+            <plugin>
+                <inherited>true</inherited>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <mainClass>servicetalk.Server</mainClass>
+                        </manifest>
+                    </archive>
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>make-assembly</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <repositories>
+        <repository>
+            <id>maven-central</id>
+            <url>https://repo1.maven.org/maven2/</url>
+        </repository>
+    </repositories>
```

Comment:
This `<repositories>` block looks unnecessary.  Does it build without this?  If so, please remove it.

Reply:
I would need to double check this tomorrow, as I believe it points to a non-default repository.